src/main/java/core/mate/academy/service/MachineServiceImpl.java

```diff
@@ -1,7 +1,38 @@
 package core.mate.academy.service;
 
-/**
- * Your implementation of MachineService.
- */
-public class MachineServiceImpl {
+import core.mate.academy.model.Machine;
+import java.util.ArrayList;
+import java.util.List;
+
+public class MachineServiceImpl implements MachineService<Machine> {
+    private static final String TYPE_NAME_BULLDOZER = "Bulldozer";
+    private static final String TYPE_NAME_EXCAVATOR = "Excavator";
+    private static final String TYPE_NAME_Truck = "Truck";
```

Comment:
Why don't just use `getClass()` and get rid of these constants? What if we want to add another inheritant of `Machine`?